maskit/examples/load_iris.py

```diff
@@ -0,0 +1,36 @@
+from pennylane import numpy as np
+from sklearn import datasets
+from maskit.examples.utils import one_hot
+
+np.random.seed(42)
```

Comment:
Yikes. ^^

Reply:
I think this should be inside the ``if __name__ == "__main__:"`` guard. Also applies to other modules doing the same.